docs/BasicCommunication/SetAppProperties/index.md

```diff
@@ -0,0 +1,107 @@
+## SetAppProperties
+
+Type
+: Function
+
+Sender
+: HMI
+
+Purpose
+: Enable/disable an application and set authentication data
+
+!!! must
+
+1. Send `SetAppProperties`(enabled=true) if webEngine app is installed.
+
+!!!
+
+!!! may
+* Request `UpdateSDL` after setting app properties to enforce SDL to perform PTU.
+* Send `SetAppProperties`(enabled=false) to add not-installed webEngine apps to SDL
+
+!!!
+
+!!! info
+
+* The OEM store uses app data in SDL Server (synchronized from SHAID) for SetAppProperties.
+* If SDL does not know the `appID` it should trigger a PTU.
+
+!!!
+
+!!!
```

Comment:
tio remove

Reply:
done in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/187/commits/33912b35cfa6c0e7fe5044685b2a1b9021816f9f